worker/src/zimfarm_worker/common/utils.py

```diff
@@ -17,8 +19,5 @@ def format_size(value: float) -> str:
 
 
 def format_key(fingerprint: str) -> str:
```

Comment:
Do you confirm this is used only for monitoring? No impact on running tasks (assuming if necessary they don't use monitoring, which is the case anyway)?

Reply:
Yes